### DISCUSSION TITLE
Sharing Fan Curves

### DISCUSSION THREAD

**Discussion by as04-git** [seerge/g-helper#1298] [Ideas]
Hey there,

First of all, this is by far the best app I've seen for laptop controls. I just wanted to suggest a feature: 

Allowing users to easily export custom fan curves to a small document which can be imported easily back within GHelper (not just for custom fan profiles, but custom fan curves for Silent, Balanced, and Turbo too).

Making this easy and allowing users to share fan profiles on Discussions here would allow everybody to collaborate and create customized fan curves that are perfect for their specific model and use fan curves that other people have created for better temperatures, performance and reliability.

I would love to see this feature implemented within GHelper as I think it could really improve the QoL of using my laptop.

P.S. I also accidentally deleted one of my custom fan curves, perhaps add a confirmation dialog to the "Factory Defaults" button?

**Comment by seerge** [comment]
@MrAS04 hello,

You already can export fan curves, by saving them from app config ``%appdata%\ghelper\config.json`` :) And import in same way, by adding / replacing config.

``Factory Defaults`` is already a button, that requires explicit action from a user, so there is no need to make UX even more complicated here.
